.github/workflows/ci.yml

```diff
@@ -162,16 +161,11 @@ jobs:
             import os
             branch = os.environ['GITHUB_REF']
             production = os.environ['PRODUCTION_BRANCH']
-            staging = os.environ['STAGING_BRANCH']
             eb_app = os.environ['EB_APP']
             eb_env_production = os.environ['EB_ENV_PRODUCTION']
-            eb_env_staging = os.environ['EB_ENV_STAGING']
             if branch == production:
               print('::set-output name=eb_app::' + eb_app)
               print('::set-output name=eb_env::' + eb_env_production)
-            elif branch == staging:
```

Comment:
hey sense check here, is this intentional

Reply:
yeap, instead of using this to deploy to staging, we're using a new workflow `deploy_staging.yml`!

don't have the braincells to combine them so leaving them apart